composer.json

```diff
@@ -25,8 +25,8 @@
     "description" : "Common libraries for Phiremock",
     "license" : "GPL-3.0-or-later",
     "require" : {
-        "php" : "^7.2|^8.0",
-        "laminas/laminas-diactoros": "^2.0",
+        "php" : "^8.1",
+        "laminas/laminas-diactoros": "^3.3",
```

Comment:
Could you write this as "^2.0|^3.0"? That way the old version is still supported for people using it.
If we change major versions of dependencies, that would need to release a major version of this library, which will lead to release a major version of phiremock-server and phiremock-client, which will lead to release a major version of the codeception module and extension. And I think we can avoid it.

Reply:
Fixed